bundler/lib/bundler/current_ruby.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative "rubygems_ext"
```

Comment:
I think `bundler/lib/bundler/dependency.rb` may have only been requiring this due to the constants that have been moved, so perhaps that require can now be removed?

Reply:
👍 I removed it from the `dependency.rb` file. There is still a usage of `Gem::Platform::RUBY` but we don't need `rubygems_ext` for that.

I also purposedly didn't add a `require_relative "current_ruby"` in the `dependency.rb` nor `dsl.rb` file (which now makes use of CurrentRuby constants), as it's already required up the stack https://github.com/rubygems/rubygems/blob/c52a74962132c54681334300c7069bb852c4419f/bundler/lib/bundler.rb#L13 
Let me know if you'd prefer adding them explicitly where needed.

Reply:
I think it's fine, thank you!